contract/config.go

```diff
@@ -14,9 +19,16 @@ const (
 	NetworkLocal Network = "local"
 )
 
-// Config defines the different settings for connecting to Ethereum on
-// different Republic Protocol networks.
 type Config struct {
+	Ethereum                republicContract.Config `json:"republic"`
```

Comment:
the json tag for this field should be `ethereum` or `config`

Reply:
This field was not modified from the [previous version](https://github.com/republicprotocol/renex-ingress-go/blob/42422c0941914ae4eb3f0385f1128d3482ea3f1d/cmd/ingress/ingress.go#L32). The only change I made was the `ApprovedTraders` field. Do you still want me to update it?

Reply:
Instead, let's just change the go file to `Republic`

Reply:
Done